hooks/gen_docs/gen_docs_cli_usage.py

```diff
@@ -21,7 +21,7 @@
         "--output",
         str(PATH_CLI_COMMANDS_DOC),
     ]
-    subprocess.run(typer_args)
+    subprocess.run(typer_args, check=True, capture_output=True)
```

Comment:
Why do we need to set these two?

Reply:
https://docs.astral.sh/ruff/rules/subprocess-run-without-check/

I thought that it's probably better that we have an explicit exception thrown upon failure.

I set `capture_output` because of the following text in `subprocess.run`'s documentation:

If check is True and the exit code was non-zero, it raises a CalledProcessError. The CalledProcessError object will have the return code in the returncode attribute, and output & stderr attributes if those streams were captured.